buildconf

```diff
@@ -99,6 +94,49 @@ else
   autoconf_flags="-f"
 fi
 
+# Check PHP build system tools such as autoconf and their versions.
+if ! test -f "$stamp"; then
+  echo "buildconf: checking installation..."
+
+  # Get minimum required autoconf version from the configure.ac file.
+  min_version=$(sed -n 's/AC_PREREQ(\[\(.*\)\])/\1/p' configure.ac)
+
+  # Check if autoconf exists.
+  ac_version=$($PHP_AUTOCONF --version 2>/dev/null|head -n 1|sed -e 's/^[^0-9]*//' -e 's/[a-z]* *$//')
+
+  if test -z "$ac_version"; then
+    echo "buildconf: autoconf not found." >&2
+    echo "           You need autoconf version $min_version or newer installed" >&2
+    echo "           to build PHP from Git." >&2
+    exit 1
+  fi
+
+  # Check autoconf version.
```

Comment:
The function `AX_COMPARE_VERSION` could be useful here.

Reply:
It might be in the m4 file but this is shell file script called prior to the autoconf call. So here the autoconf is not done yet. 

Yes, it might look like a bit of an over-engineering part for creating only the configure script but not done by me. It was in the php before and actually gives a bit nicer info to the user trying to install php...

About the AX_COMPARE_VERSION macro, we would need to include this in:
https://github.com/autoconf-archive/autoconf-archive/blob/master/m4/ax_compare_version.m4

If there will be need for this in the future, we can check it out...

Reply:
Of course. Totally missed that this script is run before calls to autotools.